angular/README.md

```diff
@@ -68,7 +68,7 @@ a set of [community contributed tutorials][comm].
  * Community/support:
    [mailing list](https://groups.google.com/a/dartlang.org/forum/#!forum/web),
    [Gitter chat room](https://gitter.im/dart-lang/angular)
- * [Logo (SVG)](https://raw.githubusercontent.com/dart-lang/logos/master/logos_and_wordmarks/angulardart-logo.svg)
```

Comment:
Actually, come to think of it, rather than a link to `raw.githubusercontent.com `, this should probably link to the repo instead: e.g., https://github.com/dart-lang/logos/tree/master/angular. In that way, the developer will have the choice of format. @kevmoo - WDYT? /cc @kwalrath